commands/host.vegman

```diff
@@ -56,18 +101,35 @@ function cmd_power_off {
     ipmitool chassis power off
     echo "Chassis power is switched off, host is down."
   else
-    ipmitool power off
+    ipmitool power soft
     echo "Shutdown signal sent to host, waiting to power the system off."
   fi
 }
 
-## @sudo cmd_power_reboot admin,operator
-## @restrict cmd_power_status admin,operator
-## @doc cmd_power_reboot
-## Performs host reboot
-#function cmd_power_reboot {
-#  not implemented
-#}
+# @sudo cmd_power_reboot admin,operator
+# @restrict cmd_power_reboot admin,operator
+# @doc cmd_power_reboot
+# Performs host reboot
+#   -y, --yes  Do not ask for confirmation
+function cmd_power_reboot {
+  local force=""
```

Comment:
will the warm reboot be implemented later or you just forgotten it?

Reply:
There is no standard way to implement warm reboot using ipmitool in particular or IPMI in general. There is no such command in the specification. That's why I removed the checks for `force` when copy-pasted the `cmd_power_off` function, but apparently I forgot to remove the `force` variable itself. Will remove it.